src/nunavut/lang/cpp/support/VariableLengthArray.hpp

```diff
@@ -0,0 +1,464 @@
+/*
```

Comment:
Shouldn't it be named `variable_length_array.hpp`?

Reply:
bah. Yeah.